### ISSUE TITLE
Render Add / Edit form in a new page

### ISSUE THREAD

**Issue by rakanalh** [qor/admin#34]
Does qor/admin support rendering add/edit forms in a new page?

Thanks

**Comment by raven-chen** [comment]
Sure. check https://doc.getqor.com/chapter2/theme.html please.

**Comment by rakanalh** [comment]
Ah i think i've asked the question in a wrong way. The current add/edit form are displayed in a sliding right sidebar right? I don't like this and i would like clicking on Add/Edit item to display the add/edit forms on their own page rather than in a sliding sidebar. Is that possible?

**Comment by raven-chen** [comment]
@rakanalh Yes, It is possible. check this https://doc.getqor.com/chapter2/theme.html, please. I'd say it is pretty difficult for you to do that even it is possible. We're developing a feature to disable the slideout, I'll let you know when it is released.

**Comment by lutfuahmet** [comment]
You can add field to Config struct for disable slideout. Because some resource need to slideout and someof them disable.
